src/EditorFeatures/CSharpTest2/Recommendations/DelegateKeywordRecommenderTests.cs

```diff
@@ -297,25 +304,25 @@ public async Task TestInMemberInitializer1()
 
         [WorkItem(538804, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/538804")]
         [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
-        public async Task TestNotInTypeOf()
+        public async Task TestInTypeOf()
         {
-            await VerifyAbsenceAsync(AddInsideMethod(
+            await VerifyKeywordAsync(AddInsideMethod(
 @"typeof($$"));
```

Comment:
interesting.  does that work at runtime?

Reply:
Gives you back `IntPtr`